### ISSUE TITLE
Defer getting TextField, JSONField

### ISSUE THREAD

**Issue by nguyenhung** [jakob/Postico#241]
I have several tables storing data in JSON fields. Each has 1MB+ in size. 
An option to defer retrieving these columns would be great.

**Comment by jakob** [comment]
Good idea. This has been suggested before for bytea fields (#57) and for geometry fields (#212).
